src/key/CLIKey.ts

```diff
@@ -34,6 +36,7 @@ export class CLIKey extends Key {
   constructor(private params: CLIKeyParams) {
     super();
     params.cliPath = params.cliPath || 'terrad';
+    params.cliPath = params.home || resolve(homedir(), '.terrad', 'config');
```

Comment:
@rtviii i mean this :) i think params.home is correct.

Reply:
@Vritra4 yeah, just noticed. late night, sorry. should be `.home` not `.cliPath` on the second line. At least that's what worked.

Reply:
See edits below :woozy_face: